src/suites/rulesets/lighthouse-api-standards.yaml

```diff
@@ -6,7 +6,7 @@ rules:
     description: Booleans should be prefixed with an auxiliary verb (such as is, has, or can).
     message: '`{{property}}` is not prefixed with an auxiliary verb (e.g. "veteran" becomes "isVeteran")'
     severity: warn
-    given: $..properties.[?(@.type === "boolean")]~
+    given: $..properties[?(@.type === "boolean")]~
```

Comment:
After some review I confirmed both syntax should be interchangeable.

Testing failed APIs against http://jsonpath.com/ with these checks confirmed neither rule is slow.

Reply:
Spectral appears to treat ".[" as a deep traversal.
https://github.com/stoplightio/spectral/issues/2489